src/Symfony/Component/Validator/Context/ExecutionContext.php

```diff
@@ -45,7 +45,7 @@ class ExecutionContext implements ExecutionContextInterface
     private mixed $root;
 
     private TranslatorInterface $translator;
-    private ?string $translationDomain;
+    private string|false|null $translationDomain;
```

Comment:
should be reverted

Reply:
Why? The translation domain needs to accepts `false` in order to correctly disable the translation